CI_JobHistory.py

```diff
@@ -93,8 +114,21 @@ def display_ci_links(config_data):
     
     return selected_config_data
 
+def list_of_strings(arg):
```

Comment:
Can we use the arg.split directly instead of this function or probably a lambda function?